src/components/multiselect.vue

```diff
@@ -440,6 +440,12 @@ const emptyMessage = computed(() => (searchValue.value === ''
         color: #666;
         font-style: italic;
       }
+      &:lang(ja), &:lang(zh) {
+        &::placeholder {
```

Comment:
Because of the use of `::placeholder` above, we can't use the mixin here. It looks like the selector has to come before `::placeholder`.